content/en/blog/_posts/2024-08-14-Kubernetes-v1-31-Release.md

```diff
@@ -0,0 +1,390 @@
+IP exhaustion has been a problem for long lived or large clusters, as admins needed to expand, shrink or even replace entirely the assigned Service CIDR range. 
+These operations were never supported natively and were performed via complex and delicate maintenance operations, often causing downtime on their clusters. This new feature allows users and cluster admins to dynamically modify Service CIDR ranges with zero downtime.
+
+For more details about this feature please visit the
+[Virtual IPs and Service Proxies](/docs/reference/networking/virtual-ips/#ip-address-objects) documentation page.
+
+This work was done as part of [KEP #1880](https://github.com/kubernetes/enhancements/issues/1880) by [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+
+### Traffic distribution for Services
+
+Traffic distribution for Services moves to beta in v1.31 and is enabled by default. 
+
+After several iterations on finding the best user experience and traffic engineering capabilities for Services networking, the SIG Networking implemented the `trafficDistribution` field in the Service specification, that serves as a guideline for the underlying implementation to consider while making routing decisions.
+It supersedes the functionality formerly provided by the `service.kubernetes.io/topology-mode` annotation and its precursor `topologyKeys` field (which has been deprecated since Kubernetes 1.21).
+
+For more details about this feature please read the
+[1.30 Release Blog](/blog/2024/04/17/kubernetes-v1-30-release/#traffic-distribution-for-services-sig-network-https-github-com-kubernetes-community-tree-master-sig-network)
+or visit the [Service](/docs/concepts/services-networking/service/#traffic-distribution) documentation page.
+
+This work was done as part of [KEP #4444](https://github.com/kubernetes/enhancements/issues/4444) by [SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+
+### Kubernetes VolumeAttributesClass ModifyVolume
+
+[VolumeAttributesClass](/docs/concepts/storage/volume-attributes-classes/) API is moving to beta in v1.31.
+The VolumeAttributesClass provides a generic,
+Kubernetes-native API for modifying dynamically volume parameters like provisioned IO.
+This allows workloads to vertically scale their volumes on-line to balance cost and performance, if supported by their provider.
+This feature had been alpha since Kubernetes 1.29.
+
+This work was done as a part of [KEP #3751](https://github.com/kubernetes/enhancements/issues/3751) and lead by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+## New features in Alpha
+
+_This is a selection of some of the improvements that are now alpha following the v1.31 release._
+    
+### New DRA APIs for better accelerators and other hardware management
```

Comment:
@pohly   can you review this section of the release blog?

Reply:
I can give the LGTM in place of @pohly (who is currently on vacation). This looks good / accurate to me.